parl/algorithms/torch/ppo.py

```diff
@@ -17,83 +17,188 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
-from torch.distributions import Normal
 from parl.utils.utils import check_model_method
+from torch.distributions import Normal, Categorical
 
 __all__ = ['PPO']
 
 
 class PPO(parl.Algorithm):
     def __init__(self,
                  model,
-                 clip_param,
-                 value_loss_coef,
-                 entropy_coef,
-                 initial_lr,
+                 clip_coef=None,
+                 vf_coef=None,
+                 ent_coef=None,
+                 start_lr=None,
                  eps=None,
                  max_grad_norm=None,
-                 use_clipped_value_loss=True):
+                 clip_vloss=True,
```

Comment:
Please use the old parameter name, or raise a warning to inform the user that the parameter has been renamed when old name is used